addons/website_forum/controllers/main.py

```diff
@@ -179,9 +179,9 @@ def forum_faq_karma(self, forum, **post):
         return request.render("website_forum.faq_karma", values)
 
     @http.route('/forum/get_tags', type='http', auth="public", methods=['GET'], website=True, sitemap=False)
-    def tag_read(self, query='', limit=25, **post):
+    def tag_read(self, query='', limit=25, forum='', **post):
```

Comment:
Just read it from the `post` kwargs? Let's not change method signature in stable
(also note that it is not `forum` but `forum_id`, i'd assume `forum` to hold a recordset)

Reply:
> Thanks for your contribution. Out of curiosity, do you know if the issue also exists in master? (And 13.0?)

@rdeodoo I tested this problem again in 13.0 and discovered that it still exists; I submitted a PR https://github.com/odoo/odoo/pull/94004 to solve it in version 13.0. Please give it a rating. Thank you very much!

Reply:
@rdeodoo Don't forget me. Please !